platform/security/core/security-core-impl/src/main/java/ddf/security/common/util/Security.java

```diff
@@ -219,23 +216,7 @@ public static PKIAuthenticationTokenFactory createPKITokenFactory() {
     }
 
     private static String getCertificatAlias() {
-        BundleContext context = getBundleContext();
-        if (context != null) {
-            ServiceReference configAdminRef = context.getServiceReference(ConfigurationAdmin.class);
-            ConfigurationAdmin configAdmin = (ConfigurationAdmin) context
-                    .getService(configAdminRef);
-            Configuration config;
-            try {
-                config = configAdmin.getConfiguration("ddf.platform.config", null);
-            } catch (IOException e) {
-                LOGGER.error("Could not get reference to configuration admin. ", e);
-                return null;
-            }
-            Dictionary<String, Object> properties = config.getProperties();
-            return (String) properties.get("host");
-        }
-        LOGGER.warn("Unable to get system certificate alias");
-        return null;
+        return System.getProperty("org.codice.ddf.system.hostname");
```

Comment:
I'm going to leave that for the other ticket. I didn't want to introduce any new dependencies with this change.